tests/test_filters.py

```diff
@@ -4,7 +4,7 @@
 from email.utils import formatdate
 
 from nyaa.routes import (_jinja2_filter_rfc822, _jinja2_filter_rfc822_es, get_utc_timestamp,
-                         get_display_time, timesince)
+                         get_display_time, timesince, filter_truthy, category_name)
```

Comment:
I know neither you nor this patch of code has anything to do with the `filter_truthy` code per se, but I thought I might bring this up now anyway since you're touching it. There's exactly zero need for `filter_truthy`, the Jinja2's `select` and `reject` filters without arguments serve literally the same purpose, a truth value test filtering on a sequence.